pipelines/environments/promptflow-runtime/spec.yaml

```diff
@@ -0,0 +1,17 @@
+$schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
+
+description: >-
+  Runtime environment for PromptFlow.
```

Comment:
we need to add an aka.ms link to changelog here.

Reply:
updated with CHANGELOG link